pkg/api/customization/pipeline/cluster_pipeline.go

```diff
@@ -269,3 +284,30 @@ func (h *ClusterPipelineHandler) authAddAccount(clusterPipeline *v3.ClusterPipel
 	}
 	return account, nil
 }
+
+func (h *ClusterPipelineHandler) getGithubConfigCR() (*v3.GithubConfig, error) {
+	authConfigObj, err := h.AuthConfigs.ObjectClient().UnstructuredClient().Get("github", metav1.GetOptions{})
```

Comment:
Why not use cache here?

Reply:
We are getting a `GithubConfig` here, which is a subtype of `AuthConfig`(and stored as an AuthConfig crd). So we need to get the unstructured object of the AuthConfig object.